packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -289,12 +290,75 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+    regularLabel: [
+      classNames.regularLabel,
+      {
+        position: 'absolute',
+
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      disabled && {
+        background: theme.palette.neutralLight,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'GrayText'
+          }
+        }
+      },
+      vertical
+        ? {
+            margin: '0 auto',
+            height: '0px',
+            marginBottom: '20px',
+            transform: getRTL() ? 'translateX(6px)' : 'translateX(-6px)'
+          }
+        : {
+            height: '0px',
```

Comment:
Height is the same in both cases so maybe hoist at top as default styles? You can even avoid using the ternary if you now that the horizontal is the default. You would have:
```ts
regularLabel: [
  ...,
  {
    height: 0,
    transform: 'translateY(6px) translateX(-6px)'
  },
  ...,
  vertical && {
    margin: '0 auto 20px',
    transform: getRTL() ? 'translateX(6px)' : 'translateX(-6px)'
  }
]

Also, just curious why do you need `translate` to position the labels?

Reply:
ok just made the changes and I added a translate because this currently places a label on the track then translates it down so its like under the track